test.js

```diff
@@ -156,9 +156,94 @@ cases = {
     '/r/very+clever+multireddit+reddit.com+t:fork+yay':
         '<p><a href="/r/very+clever+multireddit+reddit.com+t:fork+yay">/r/very+clever+multireddit+reddit.com+t:fork+yay</a></p>\n',
 
+    '/r/all-minus-something':
+        '<p><a href="/r/all-minus-something">/r/all-minus-something</a></p>\n',
+
+    '/r/notall-minus':
+        '<p><a href="/r/notall">/r/notall</a>-minus</p>\n',
```

Comment:
`/r/all-` is a special case for the feature that allows you to filter certain subreddits out of `/r/all`. `-` in an old-style multireddit doesn't mean anything outside of that case (`/r/notall-minus` would mean get a listing from `/r/notall` while filtering out results from `/r/minus`,) so we don't autolink it. [Snudown has the same behaviour](https://github.com/reddit/snudown/blob/master/test_snudown.py#L163).